js/ui/windowManager.js

```diff
@@ -37,10 +37,40 @@ const DIM_BRIGHTNESS = -0.3;
 const DIM_TIME = 0.500;
 const UNDIM_TIME = 0.250;
 const SKYPE_WINDOW_CLOSE_TIMEOUT_MS = 1000;
+const ICON_BOUNCE_MAX_SCALE = 0.4;
+const ICON_BOUNCE_ANIMATION_TIME = 1.0;
+const ICON_BOUNCE_ANIMATION_TYPE_1 = 'easeOutSine';
+const ICON_BOUNCE_ANIMATION_TYPE_2 = 'easeOutBounce';
 
 const DISPLAY_REVERT_TIMEOUT = 30; // in seconds - keep in sync with mutter
 const ONE_SECOND = 1000; // in ms
 
+function animateBounce(actor)
+{
+        Tweener.removeTweens(actor);
+        if (!Tweener.isTweening(actor)) {
+            Tweener.addTween(actor, {
+                scale_y: 1 + ICON_BOUNCE_MAX_SCALE,
+                scale_x: 1 + ICON_BOUNCE_MAX_SCALE,
+                translation_y: actor.height * ICON_BOUNCE_MAX_SCALE,
+                translation_x: actor.width * ICON_BOUNCE_MAX_SCALE / 2,
+                time: ICON_BOUNCE_ANIMATION_TIME * 0.25,
+                delay: 0.3,
+                transition: ICON_BOUNCE_ANIMATION_TYPE_1
+            });
+            Tweener.addTween(actor, {
+                scale_y: 1,
+                scale_x: 1,
+                translation_y: 0,
+                translation_x: 0,
+                time: ICON_BOUNCE_ANIMATION_TIME * 0.75,
+                transition: ICON_BOUNCE_ANIMATION_TYPE_2,
+                delay: ICON_BOUNCE_ANIMATION_TIME * 0.25 + 0.3,
+                onComplete: function() {animateBounce(actor);}
```

Comment:
Does it make sense to call `animateBounce` again with the same actor? Seems like this function is just called as soon as the animation finishes?